templates/components/cart/shipping-estimator.html

```diff
@@ -34,14 +34,18 @@
             </dt>
 
             <dd class="estimator-form-input">
-                <select class="form-select" id="shipping-state" name="shipping-state" data-field-type="State">
-                    <option>{{lang 'cart.shipping_estimator.select_a_state'}}</option>
-                    {{#each states}}
-                        <option value="{{id}}" {{#if selected}}selected="selected"{{/if}}>
-                            {{name}}
-                        </option>
-                    {{/each}}
-                </select>
+                {{#if selected_state}}
```

Comment:
Yeah I just check the context. `selected_state` could be null while `states` is an array with all the states